### PR TITLE
ninja: add job limit example

### PR DESCRIPTION
**Pull request by cyqsimon**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 1.11.1**

I used the "at max" wording because `ninja` runs with `$(nproc) + 2` jobs by default (as opposed to `make`, which runs 1 job by default), so `-j` in this case is more of a restrictive flag.

### CODE REVIEW

File: pages.zh/common/ninja.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `ninja`
 
+- 在当前目录下构建，最多并行执行 4 个作业：
+
+`ninja -j{{4}}`
```
Comment (kbdharun) [inline pages.zh/common/ninja.md:12]:
```suggestion
`ninja -j {{4}}`
```

File: pages/common/ninja.md [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 `ninja`
 
+- Build in the current directory, with 4 parallel jobs at max:
+
+`ninja -j{{4}}`
```
Comment (kbdharun) [inline pages/common/ninja.md:12]:
```suggestion
`ninja -j {{4}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Ref. 

```powershell
PS C:\Users\kbdha> ninja --help
usage: ninja [options] [targets...]

if targets are unspecified, builds the 'default' target (see manual).

options:
  --version      print ninja version ("1.11.1.git.kitware.jobserver-1")
  -v, --verbose  show all command lines while building
  --quiet        don't show progress status, just command output

  -C DIR   change to DIR before doing anything else
  -f FILE  specify input build file [default=build.ninja]

  -j N     run N jobs in parallel (0 means infinity) [default=14 on this system]
  -k N     keep going until N jobs fail (0 means infinity) [default=1]
  -l N     do not start new jobs if the load average is greater than N
  -n       dry run (don't run commands but act like they succeeded)

  -d MODE  enable debugging (use '-d list' to list modes)
  -t TOOL  run a subtool (use '-t list' to list subtools)
    terminates toplevel options; further flags are passed to the tool
  -w FLAG  adjust warnings (use '-w list' to list warnings)
```

**Review by kbdharun — APPROVED** [review]

File: pages/common/ninja.md [hunk 3]
```diff
@@ -7,6 +7,10 @@
 
 `ninja`
 
+- Build in the current directory, with 4 parallel jobs at max:
```
Comment (acuteenvy) [inline pages/common/ninja.md:10]:
```suggestion
- Build in the current directory, executing 4 jobs at a time in parallel:
```

**Review by kbdharun — APPROVED** [review]
LGTM

@blueskyson or @cyqsimon Any updated suggestions for the Chinese translation of `ninja`?

**Review by einverne — APPROVED** [review]
LGTM.

**Pull request merged**